doc/src/compiler.md

```diff
@@ -875,7 +875,7 @@ Other complex types are:
 
     ```javascript
     {
-      "type": "constant,
+      "type": constant,
```

Comment:
It's actually `"type": "constant"` (both are strings).

Reply:
Ok.